source/base/Text.ooc

```diff
@@ -53,6 +53,30 @@ Text: cover {
 	operator != (other: String) -> Bool { !(this == other) }
 	operator != (other: This) -> Bool { !(this == other) }
 	operator + (other: This) -> This { This new(this _buffer + other _buffer) }
+	operator + (other: Int) -> This {
+		converted := other toString()
```

Comment:
It's the same reason we can't have `toText` in `Cell` right now. These files are in the sdk, and cannot depend on things outside the sdk. Once we've got `Text` inside the sdk, it's another story.